installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -1,11 +1,25 @@
 ## Migrating from Appliances
 
 ### Notes
-- Current architecture in podified limits us to running a single server and zone in a Kubernetes namespace.
+
+- Multi-server / multi-zone:
+  Current architecture in podified limits us to running a single server and zone in a Kubernetes namespace.
```

Comment:
I think technically we do have multi-zone - it can be configured via node labels.  I don't remember the specifics, so we'd have to dig in.